pkg/kritis/cryptolib/public_key.go

```diff
@@ -39,16 +40,18 @@ type PublicKey struct {
 	ID string
 }
 
-// NewPublicKey creates a new PublicKey. `keyType` contains the type of the
-// public key, one of Pgp, Pkix or Jwt. `keyData` contains the raw key
-// material. `keyID` contains a unique identifier for the public key. For PGP,
-// this field should be left blank. The ID will be the OpenPGP RFC4880 V4
-// fingerprint of the key. For PKIX and JWT, this may be left blank, and the ID
-// will be generated based on the DER encoding of the key. If not blank, the ID
-// should be a StringOrURI: it must either not contain ":" or be a valid URI.
-func NewPublicKey(keyType KeyType, keyData []byte, keyID string) (*PublicKey, error) {
+// NewPublicKey creates a new PublicKey.
+// `authType` indicates the transport format of the Attestation this PublicKey
+// verifies, one of Pgp, Pkix or Jwt.
+// `keyData` contains the raw key material.
+// `keyID` contains a unique identifier for the public key. For PGP, this field
+// should be left blank. The ID will be the OpenPGP RFC4880 V4 fingerprint of
+// the key. For PKIX and JWT, this may be left blank, and the ID  will be
+// generated based on the DER encoding of the key. If not blank, the ID should
+// be a StringOrURI: it must either not contain ":" or be a valid URI.
+func NewPublicKey(authType AuthenticatorType, keyData []byte, keyID string) (*PublicKey, error) {
 	newKeyID := ""
-	switch keyType {
+	switch authType {
```

Comment:
nit: authType -> authenticatorType, because auth is generally used to abbreviate authentication, not authenticator

Reply:
Done.